### ISSUE TITLE
Add an option to change hotkey

### ISSUE THREAD

**Issue by iambipinpaul** [spectresystems/jarvis#23]
I use  ALT + Space to open CMD in visual Studio solutions explore selected directory,so it will be nice to have an option to changes that.

**Comment by stephanjohnson** [comment]
HA! I am so glad I'm not the only one to have this exact problem. I use Mads Kristensen's Open Command Line Visual Studio extension (https://marketplace.visualstudio.com/items?itemName=MadsKristensen.OpenCommandLine) and I've grown accustomed to using ALT + Space to open a command prompt from Visual Studio.

It would be great if you could change the hotkey

**Comment by patriksvensson** [comment]
Dependent on #6
